examples/simple-auth/src/components/Header/header.module.css

```diff
@@ -31,7 +31,7 @@
   text-decoration: none;
 }
 
-.header__link--home {
+.header__linkHome {
```

Comment:
Just checking - this is not invalid per se - it just convert to `header__linkHome`?

Reply:
the `--modifier` is convention from BEM ( http://getbem.com/naming/ ).

I just checked and it does wart but unfortunately is renamed as mentioned above, so I guess it make sense to rename it here so export name actually match class name (just a bit unfortunate :/)